website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -60,6 +54,12 @@ Before continuing further, follow the [testing guide](/contributing/release-test
 
 <AsyncTestingNote/>
 
+### 4. Bump monorepo packages
+
+- Update monorepo package dependencies by running `yarn bump-all-updated-packages`. Bear in mind that all the package bumps must be on patch level, and be on the minor you are working on. Read more about this script and how it works [here](./release-updating-packages).
```

Comment:
Would you add an example here to avoid any confusion?

Reply:
Yeah, I can improve the phrasing + add an example command 👍🏻.

Sidenote: We already have published 0.74.0 for monorepo dependencies (cc @lunaleaps). This means that, keeping this guidance, for at least the 0.74 release cycle we can't have matching versions of dependencies per `react-native` release 🫤.

<img width="880" alt="image" src="https://github.com/facebook/react-native-website/assets/2547783/24d469a0-9474-4097-a576-4b42e4087912">

Reply:
@lunaleaps Let's also update step 12 in this guide to so that we `set-version` to `0.xx-main` going forward.